sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,29 +1,41 @@
 # Release History
 
-## 2.0.0-beta.1 (Unreleased)
+## 2.0.0-beta.1 (2021-03-23)
 
 This release continues with the changes from `1.2.4` and `1.2.4-beta.1`.
 
+### Breaking changes
+
 - The `getToken` methods will now never return `null`. If a token is not available, we will return a rejected promise.
-- `DefaultAzureCredential`'s implementation for browsers was simplified to throw a simple error instead of trying credentials that were already not supported for the browser.
-- Breaking Change: `InteractiveBrowserCredential` for the browser now requires the client ID to be provided.
-- Documentation was added to elaborate on how to configure an AAD application to support `InteractiveBrowserCredential`.
-- Replaced the use of the 'express' module with a Node-native http server, shrinking the resulting identity module considerably
-- Updated `@azure/msal-node-extensions` to [1.0.0-alpha.6](https://www.npmjs.com/package/@azure/msal-node-extensions/v/1.0.0-alpha.6).
+- `InteractiveBrowserCredential` for the browser now requires a client ID to be provided.
```

Comment:
Can we add a one liner as to why client id is so important that we made it mandatory?

Reply:
I'll push this:

> After we identified that the default client ID used for Node JS was not viable for the browser, we've made the client ID parameter required on the browser version of `InteractiveBrowserCredential`.

Reply:
sounds good
Nit: Use `Node.js` instead of `Node JS` as per the way it is branded. For example, see https://nodejs.org/en/